lib/balance-tx/test/data/balanceTx/conway/pingPong_1/golden

```diff
@@ -0,0 +1,81 @@
+ 1.750000,ErrBalanceTxAssetsInsufficient (ErrBalanceTxAssetsInsufficientError {available = MaryValue (Coin 1750000) (MultiAsset (fromList [])), required = MaryValue (Coin 2000000) (MultiAsset (fromList [])), shortfall = MaryValue (Coin 250000) (MultiAsset (fromList []))})
+ 1.800000,ErrBalanceTxAssetsInsufficient (ErrBalanceTxAssetsInsufficientError {available = MaryValue (Coin 1800000) (MultiAsset (fromList [])), required = MaryValue (Coin 2000000) (MultiAsset (fromList [])), shortfall = MaryValue (Coin 200000) (MultiAsset (fromList []))})
+ 1.850000,ErrBalanceTxAssetsInsufficient (ErrBalanceTxAssetsInsufficientError {available = MaryValue (Coin 1850000) (MultiAsset (fromList [])), required = MaryValue (Coin 2000000) (MultiAsset (fromList [])), shortfall = MaryValue (Coin 150000) (MultiAsset (fromList []))})
+ 1.900000,ErrBalanceTxAssetsInsufficient (ErrBalanceTxAssetsInsufficientError {available = MaryValue (Coin 1900000) (MultiAsset (fromList [])), required = MaryValue (Coin 2000000) (MultiAsset (fromList [])), shortfall = MaryValue (Coin 100000) (MultiAsset (fromList []))})
+ 1.950000,ErrBalanceTxAssetsInsufficient (ErrBalanceTxAssetsInsufficientError {available = MaryValue (Coin 1950000) (MultiAsset (fromList [])), required = MaryValue (Coin 2000000) (MultiAsset (fromList [])), shortfall = MaryValue (Coin 50000) (MultiAsset (fromList []))})
+ 2.000000,ErrBalanceTxUnableToCreateChange (ErrBalanceTxUnableToCreateChangeError {requiredCost = Coin 170693, shortfall = Coin 170693})
+ 2.050000,ErrBalanceTxUnableToCreateChange (ErrBalanceTxUnableToCreateChangeError {requiredCost = Coin 170693, shortfall = Coin 120693})
+ 2.100000,ErrBalanceTxUnableToCreateChange (ErrBalanceTxUnableToCreateChangeError {requiredCost = Coin 170693, shortfall = Coin 70693})
+ 2.150000,ErrBalanceTxUnableToCreateChange (ErrBalanceTxUnableToCreateChangeError {requiredCost = Coin 170693, shortfall = Coin 20693})
+ 2.200000,0.200000,0.167261
+ 2.250000,0.250000,0.167261
+ 2.300000,0.300000,0.167261
+ 2.350000,0.350000,0.167261
+ 2.400000,0.400000,0.167261
+ 2.450000,0.450000,0.167261
+ 2.500000,0.500000,0.167261
+ 2.550000,0.550000,0.167261
+ 2.600000,0.600000,0.167261
+ 2.650000,0.650000,0.167261
+ 2.700000,0.700000,0.167261
+ 2.750000,0.750000,0.167261
+ 2.800000,0.800000,0.167261
+ 2.850000,0.850000,0.167261
+ 2.900000,0.900000,0.167261
+ 2.950000,0.950000,0.167261
+ 3.000000,1.000000,0.167261
+ 3.050000,1.050000,0.167261
+ 3.100000,1.100000,0.167261
+ 3.150000,1.150000,0.167261
+ 3.200000,0.170121,0.170121
+ 3.250000,0.170121,0.170121
+ 3.300000,0.170121,0.170121
+ 3.350000,0.170121,0.170121
+ 3.400000,0.170121,0.170121
+ 3.450000,0.170121,0.170121
+ 3.500000,0.170121,0.170121
+ 3.550000,0.170121,0.170121
+ 3.600000,0.170121,0.170121
+ 3.650000,0.170121,0.170121
+ 3.700000,0.170121,0.170121
+ 3.750000,0.170121,0.170121
+ 3.800000,0.170121,0.170121
+ 3.850000,0.170121,0.170121
+ 3.900000,0.170121,0.170121
+ 3.950000,0.170121,0.170121
+ 4.000000,0.170121,0.170121
```

Comment:
A bit more than for babbage: 0.169857

A con with separating the goldens by era like this is that we won't ever get GH diffs between eras. Not unless we add `latest/pingPong_1` separately from `conway/pingPong_1` and `babbage/pingPong_1`, which I concidered but didn't end up doing now.

Reply:
> A con with separating the goldens by era like this is that we won't ever get GH diffs between eras. 

🤷 I think that's fine.